kclvm/ast/src/ast.rs

```diff
@@ -998,6 +1012,21 @@ pub struct StringLit {
     pub value: String,
 }
 
+/// Generate ast.StringLit from String
+impl TryFrom<String> for StringLit {
+    type Error = &'static str;
+
+    fn try_from(value: String) -> Result<Self, Self::Error> {
+        Ok(
+            Self{
+                value: value.clone(),
+                raw_value: value,
```

Comment:
`raw_value` maybe not same with the value. For example:
`raw_value` is `"123"` and `value` is `123`

Reply:
You can write raw_value with `format!("{:?}", value)`